src/components/parameter.jsx

```diff
@@ -30,6 +30,12 @@ class Parameter extends React.Component {
             ? <p className="description">{parameter.description}</p>
             : <em><FormattedMessage id="noDescription"/></em>
           }
+          { parameter.documentation &&
+            (<span><FormattedMessage id="documentationText"/></span>)
+          }
+          { parameter.documentation &&
+            (<p className="documentation">{ parameter.documentation.trim() }</p>)
```

Comment:
The trimming is not necessary if we [clean the values on the API side](https://github.com/openfisca/openfisca-core/pull/717/files#r219781241) :)

If the first use of this new field need trimming, this _should_ probably be done on the API side.